docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -160,7 +160,65 @@ For example, you could use interpolation to build the user's full name:
 }
 ```
 
-Interpolated shortcode substituted values will always be casted to strings. For example, if the user does not have a last name associated, the above full name value would be "null John".
+Interpolated shortcodes will always result to string values. For example, if the user does not have a last name associated, the above full name value would be "null John".
+
+### Coalescing
+
+Shortcode coalescing is a conditional expression of the form:
+
+```json
+"{{ <operand_1> || <operand_2> || <operand_n> }}"
+```
+
+For example, given the following template:
+
+```json
+{
+  "has_verified_contact_info": "{{user.has_verified_email || user.has_verified_phone}}",
+
+  // fallback to a string value
+  "full_name": "{{user.first_name || 'John'}} {{user.last_name || 'Doe' }}",
+
+  // fallback to a number value
+  "age": "{{user.public_metadata.age || user.unsafe_metadata.age || 30 }}"
+}
+```
+
+The output after substitution (subject to the actual user data) would look like this:
```

Comment:
❓ We could perhaps describe the state more clearly here as well and avoid the parenthesized clarification?

Reply:
Addressed https://github.com/clerkinc/clerk-docs/pull/412#discussion_r1360366045